data/campaigns/Northern_Rebirth/scenarios/02_01_Infested_Caves.cfg

```diff
@@ -629,16 +677,44 @@
                     equals=6
                 [/variable]
                 [then]
-                    [message]
-                        speaker=unit
-                        message= _ "On the end of the passage is another door. I try to open it."
-                    [/message]
+                    [if]
+                        [variable]
+                            name=unit.id
+                            equals=Tallin
+                        [/variable]
+                        [then]
+                            [message]
+                                speaker=Tallin
+                                message= _ "Another gate... I guess I'll kick it down like the other one."
+                            [/message]
+                        [/then]
+                        [else]
+                            [message]
+                                speaker=unit
+                                message= _ "On the end of the passage is another gate. I will try to open it."
+                            [/message]
+                        [/else]
+                    [/if]
                 [/then]
                 [else]
-                    [message]
-                        speaker=unit
-                        message= _ "Here is a similar door as the one before. I try to open it."
-                    [/message]
+                    [if]
+                        [variable]
+                            name=unit.id
+                            equals=Tallin
+                        [/variable]
+                        [then]
+                            [message]
+                                speaker=Tallin
+                                message= _ "Another one. I'll kick this one down too."
+                            [/message]
+                        [/then]
+                        [else]
+                            [message]
+                                speaker=unit
+                                message= _ "Here is a similar gate as the one before. I will try to open it."
```

Comment:
I recognise this is from the the original text but the first sentence sounds odd to me. I'd prefer 'to' rather than 'as' (this is master, not stable) but if people prefer to leave it as-is, so be it.

Reply:
I also felt it was off, so I ended up changing it.